Doc/library/__main__.rst

```diff
@@ -1,25 +1,186 @@
-
-:mod:`__main__` --- Top-level script environment
-================================================
+:mod:`__main__` --- CLIs, import-time behavior, and ``__name__ == '__main__'``
+==============================================================================
 
 .. module:: __main__
-   :synopsis: The environment where the top-level script is run.
+   :synopsis: CLIs, import-time behavior, and ``__name__ == '__main__'``
```

Comment:
I think the synopsis should explain a little more than the title (for example, by expanding the CLI abbreviation)

Reply:
Done; see 7fe7f1c170077ac63d94f93c67333454c2681216